DTXMania/Code/Stage/04.Config/CActConfigList.cs

```diff
@@ -738,14 +725,13 @@ public void tSetupItemList_Drums()
             this.iDrumsHAZARD = new CItemToggle("HAZARD", CDTXMania.ConfigIni.bHAZARD,
                 "ドSハザードモード\n" +
                 "GREAT以下の判定でも回数が減ります。",
-                "SuperHazardMode\n" +
-                "");
+                "Turn ON to decrease the damage gauge if the judgment\nis anthing less than PERFECT. Good Luck!",
             this.listItems.Add(this.iDrumsHAZARD);
 
             this.iDrumsTight = new CItemToggle("Tight", CDTXMania.ConfigIni.bTight,
                 "ドラムチップのないところでパッドを\n" +
                 "叩くとミスになります。",
-                "Hitting pad without chip is a MISS.");
+				"Any pad hits not linked to a note\nare graded as a MISS.",
```

Comment:
```suggestion
		"Any pad hits not linked to a note\nare graded as a MISS.",
```
Copy-paste alignment issue.